src/java/org/apache/cassandra/repair/RepairRunnable.java

```diff
@@ -840,23 +794,58 @@ public void runMayThrow() throws Exception
         }
     }
 
-    private static final class NeighborsAndRanges
+    static final class NeighborsAndRanges
     {
         private final boolean force;
         private final Set<InetAddressAndPort> allNeighbors;
```

Comment:
nit: Would it make things clearer to name `allNeighbors` `liveEndpoints`? (Is there ever a time when it's not the set of live endpoints?)

Reply:
At least in the current code base, the `allNeighbors` are the live ones. 
Given that, maybe rename the class `NeighborsAndRanges` `LiveNeighborsAndRanges`? Only having `liveEndpoints` implies there could be a set of dead endpoints in the neighbors.

Reply:
I'm fine wither either one.